tkg/clusterclient/clusterclient.go

```diff
@@ -2697,18 +2700,15 @@ func (c *client) RemoveMatchingLabelsFromResources(gvk schema.GroupVersionKind,
 	if err != nil {
 		return err
 	}
-
+	var removePatches []string
+	for _, key := range keysToRemove {
+		removePatches = append(removePatches, fmt.Sprintf("%q: null", key))
```

Comment:
Could you help me to understand how to remove key:value by using patch here: you set "key:null", and then the merge patch will remove these keys with null values automatically?

Reply:
Yes, Please refer to the kubectl command 
`kubectl patch PackageInstall core-management-plugins -n tkg-system --type=merge -p '{"metadata": {"labels": {"kapp.k14s.io/app": null, "kapp.k14s.io/association": null}}}'`